Examples/Ex00_properties.m

```diff
@@ -2,6 +2,7 @@
 %* HGS 2.0 (adapted from the original HGS 1.3) 
 %* Original by Arnau Miro, Pau Manent, Manel Soria 
 %* Adapted by Caleb Fuster
+%* Commented by Paulino Gil -> Only this comment

```

Comment:
Paulino, this is the hall of fame. You should contribute a more significant change to be there.
But don't be disappointed and keep trying.

Reply:
Sure ! You're right...I hope I will not disappoint you.